.circleci/publish_to_github.sh

```diff
@@ -83,5 +88,5 @@ publishPackage() {
   fi
 }
 
-# Run the publish package function
-publishPackage
+publishPackage "@angular/build-tooling" "dev-infra-private-build-tooling-builds" "dist/bin/npm_package"
```

Comment:
Should we change this to script taking in the parameters? and then call the specifics from the config.yml file?

Reply:
That seems like a nice idea, will do